config.go

```diff
@@ -971,20 +957,15 @@ func createDefaultConfigFile() error {
 	}
 	defer src.Close()
 
-	destinationPath := filepath.Join(dcrdHomeDir, defaultConfigFilename)
-	err = os.MkdirAll(dcrdHomeDir, 0700)
-	if err != nil {
-		return err
-	}
-	dest, err := os.OpenFile(destinationPath, os.O_RDWR|os.O_CREATE|os.O_TRUNC,
-		0644)
+	dest, err := os.OpenFile(destinationPath,
+		os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0700)
```

Comment:
`0700` is definitely not right.  It's a config file, it doesn't need to be executable.  It's debatable whether you want to do `0600`.